main/com.spotify/eval/scenarios.txt

```diff
@@ -256,8 +256,9 @@ A: >> expecting = null
 U: \t $dialogue @org.thingpedia.dialogue.transaction.execute;
 U: now => (@com.spotify.playable(), release_date >= makeDate(2000, 1, 1) && release_date <= makeDate(2000, 1, 1) + 1year) => notify;
 
-A: I (see|have( found)?)(one, it is)? Yellow. It is( a)?( pop)? music by Coldplay( released in July 9, 2000)?\. Would you like to play it on Spotify\?
-A: >> expecting = command
+A: I (see|have( found)?) Yellow and The Real Slim Shady\. Yellow is a music by Coldplay, while The Real Slim Shady Eminem wrote\.
```

Comment:
We're doing poetry, I see. File this as a bug in genie-toolkit, the priority of the different parts of speech is messed up if we generate this.